src/api/app/controllers/webui/package_controller.rb

```diff
@@ -798,11 +798,11 @@ def live_build_log
     end
 
     begin
-      @package = Package.get_by_project_and_name(params[:project], params[:package],
+      @package = Package.get_by_project_and_name(@project.to_param, params[:package],
```

Comment:
There is no need to do [.to_param](http://apidock.com/rails/ActiveRecord/Integration/ClassMethods/to_param). This is the default for generating urls.